app/models/game.rb

```diff
@@ -1,5 +1,6 @@
 class Game < ActiveRecord::Base
-	belongs_to :user
+	belongs_to :white_player, class_name: "User"
+	belongs_to :black_player, class_name: "User"
 	has_many :pieces
```

Comment:
quick question Aaron - Anna and I had discussed whether it should be belongs_to or has_many. In  our user.rb model we put  
class User < ActiveRecord::Base 
has_many :games

So doesn't this mean we have to have 
class Game<ActiveRecord::Base
belongs_to :user 
?

I remember reading that has_many and belongs_to should be paired, but I can't find it.